<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wifi</class>
 <widget class="QMainWindow" name="wifi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#wifi
{
border-image: url(:/t507_button_image/all/3.webp);
}
/****************按钮部件*******************/
QPushButton{
     background-color: rgba(100, 225, 100, 120);/*背景色*/
     border-style: outset; /*边框样式*/
     border-width:  2px;  /*边框宽度为4像素*/
     border-radius: 10px; /*边框圆角半径*/
     border-color: rgba(255, 225, 255, 30);/*边框颜色*/
     font: bold 18px;/*字体*/
     color:rgba(0, 0, 0, 100);/*字体颜色*/
     padding: 6px; /*填衬*/
	outline: none; /*去掉按钮的虚线框*/
}

QPushButton:hover{ /*鼠标悬停在按钮上时*/
        background-color:rgba(100,255,100, 100);
        border-color: rgba(255, 225, 255, 200);
        color:rgba(0, 0, 0, 200);
}

QPushButton:pressed { /*按钮被按下时*/
     background-color:rgba(100,255,100, 200);
     border-color: rgba(255, 225, 255, 30);
     border-style: inset;
     color:rgba(0, 0, 0, 100);
}

/****************滑块部件*******************/


QSlider::handle:horizontal { /*水平滑块的手柄*/
        image: url(:/image/sliderHandle.png);
 }

QSlider::sub-page:horizontal { /*水平滑块手柄以前的部分*/
        border-image: url(:/image/slider.png);/*边框图片*/
 }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ReturnBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="translation.qrc">
      <normaloff>:/t507_button_image/return.png</normaloff>:/t507_button_image/return.png</iconset>
    </property>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>231</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>235</height>
      </rect>
     </property>
     <attribute name="label">
      <string>wifi</string>
     </attribute>
     <widget class="QPushButton" name="WifiConnectBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>wifi connect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="WifiQualityBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>signal quality</string>
      </property>
     </widget>
     <widget class="QPushButton" name="WifiStatusBtn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>status</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>235</height>
      </rect>
     </property>
     <attribute name="label">
      <string>hotspot</string>
     </attribute>
     <widget class="QPushButton" name="HotspotConBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>hotspot connect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="HotspotStatusBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>status</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>20</width>
      <height>601</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>761</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="wifihomepage">
     <widget class="QListWidget" name="WifiListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>731</width>
        <height>471</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="SwitchControl" name="WifiSwitch" native="true">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>30</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="RefreshBtn">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>20</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>refresh</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="WifiStatusPag">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>wifi status</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="WifiText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>711</width>
        <height>381</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="HotspotPag">
     <widget class="QLineEdit" name="HtName">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>190</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="HtPasswd">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>290</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="HotspotName">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>name</string>
      </property>
     </widget>
     <widget class="QLabel" name="HotspotPasswd">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>290</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>passwd</string>
      </property>
     </widget>
     <widget class="QPushButton" name="HotspotBuildBtn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>390</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>up</string>
      </property>
     </widget>
     <widget class="QPushButton" name="HotspotDownBtn">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>390</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>down</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>hot spot</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="WifiExistPag">
     <widget class="QPushButton" name="WifiExistRemoveBtn">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>230</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Network</string>
      </property>
     </widget>
     <widget class="QPushButton" name="WifiBackBtn">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>450</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="WifiActiveBtn">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>110</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Join Network</string>
      </property>
     </widget>
     <widget class="QLabel" name="WifiInfoNameLab">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>251</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>wifiname</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>security</string>
      </property>
     </widget>
     <widget class="QLabel" name="WifiInfoSecurityLab">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>170</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="ChangePasswdBtn">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>340</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Password</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchControl</class>
   <extends>QWidget</extends>
   <header location="global">switchcontrol.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="translation.qrc"/>
 </resources>
 <connections/>
</ui>
